content/docs/tasks/traffic-management/app-health-check/index.md

```diff
@@ -76,19 +92,11 @@ This section shows how to configure health checking with the HTTP request option
 
 ### Mutual TLS is disabled
 
-Run this command to remove the mutual TLS policy:
+Run this command to remove the mutual TLS policy and corresponding destination rule:
```

Comment:
See my previous comment.
These task would contain two steps, one for each command:

1. To remove remove the mutual TLS policy, run:

    {{< text bash >}}
    $ kubectl delete policies default
    {{< /text >}}

1. To remove remove the corresponding destination rule, run:

    {{< text bash >}}
    $ kubectl delete destinationrules default
    {{< /text >}}

Reply:
Done